### PR TITLE
fix : added responsiveness and readme file

### PR DESCRIPTION
**Pull request by someshtiwarii**
- made navbar responsive
- made product-listing page responsive
- made cart page responsive
- made wishlist page responsive
- made login and signup pages responsive
- added github documentation

### CODE REVIEW

File: css/general.css [hunk 1]
```diff
@@ -50,4 +50,111 @@ body {
...
 .search-container ::placeholder{
     color:white;
+}
+
+
+
+@media only screen and (max-width: 860px){
+
+    .navbar {
+        padding: 0.5rem 0.5rem;
```
Comment (msdeshmukh009) [inline css/general.css]: You can use `padding:0.5rem;`

File: css/general.css [hunk 2]
```diff
@@ -50,4 +50,111 @@ body {
...
+    .search-container{
+        width:3.5rem;
+    }
+
+    .search-input{
+        margin-left:0.25rem;
+    }
+
+    .navbar{
+        padding: 0.2rem 0.1rem;
```
Comment (msdeshmukh009) [inline css/general.css:136]: Try to apply padding and margin in multiples of 4px/0.25rem or whatever you prefer but try to keep it consistent.

File: css/styles.css [hunk 3]
```diff
@@ -125,6 +125,144 @@
...
+    .banner-content button {
+        font-size: 80%;
+    }
+
+    .banner-content{
+        left:5%;
+    }
+
+    .featured {
+        margin: 0.5rem 0.5rem;
```
Comment (msdeshmukh009) [inline css/styles.css]: `margin:0.5rem;`

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking fine but try to keep margin and padding consistent or a better option is to create variables for it. Also one more thing the product listing page is not responsive as you mentioned. So you might want to check these things out.

**Comment by RahulKuvlekar** [comment]
Everything looks good to me

File: pages/login/login.css [hunk 4]
```diff
@@ -74,4 +74,26 @@ main {
 
 .message {
     display: none;
+}
+
+
+@media only screen and (max-width: 540px) {
+
+    .login-section {
+        padding: 2rem 0rem;
```
Comment (AbhishekkGautam) [inline pages/login/login.css]: We can just put 0 instead of 0 rem.

**Review by AbhishekkGautam — APPROVED** [review]
Everything else looks good to me.

File: pages/login/login.css (hunk 4 above)
Comment (someshtiwarii) [reply pages/login/login.css]: ok Abhishek I will do that. Thankyou :)

File: css/styles.css (hunk 3 above)
Comment (someshtiwarii) [reply css/styles.css]: Thankyou for pointing it out Mahesh, will do that

File: css/general.css (hunk 2 above)
Comment (someshtiwarii) [reply css/general.css:136]: Yes Mahesh will keep in mind from next time, thankyou :)

File: css/general.css (hunk 1 above)
Comment (someshtiwarii) [reply css/general.css]: Yes will do that

**Pull request merged**